.circleci/config.yml

```diff
@@ -98,6 +98,16 @@ jobs:
           command: |
             source py3-lint-env/bin/activate
             black --check --diff cla_public
+  webhint:
+    docker:
+      - image: node:15.0.0-stretch
```

Comment:
Sorry. Me again! We were going to try one of the official `circle/` images, weren't we? Cos we figured that would be more ready to get working here (it shouldn't have any of the perms issues we had fun solving last week, for example)